dev/plugins/hu.elte.txtuml.api.model/src/hu/elte/txtuml/api/model/Action.java

```diff
@@ -50,6 +50,7 @@
 	 * @throws NullPointerException
 	 *             if <code>classType</code> is <code>null</code>
 	 */
+	@ExternalBody
```

Comment:
@devaigergely81, according to the commit message of 84b1eca,

> This is to emphasize which methods of the API should be used in the model and which not.

Is this a valid reason?